src/arch/ArchHooks/ArchHooks_Win32.cpp

```diff
@@ -150,26 +150,22 @@ void ArchHooks_Win32::SetTime( tm newtime )
 
 void ArchHooks_Win32::BoostPriority()
 {
-	/* We just want a slight boost, so we don't skip needlessly if something happens
-	 * in the background.  We don't really want to be high-priority--above normal should
-	 * be enough.  However, ABOVE_NORMAL_PRIORITY_CLASS is only supported in Win2000
-	 * and later. */
-#ifndef ABOVE_NORMAL_PRIORITY_CLASS
-#define ABOVE_NORMAL_PRIORITY_CLASS 0x00008000
+	/* Enforcing HIGH_PRIORITY_CLASS is best for Win10+ where Windows Update can interrupt the game
+	 * and it also prevents streaming apps from taking rendering priority over Stepmania */
+#ifndef HIGH_PRIORITY_CLASS
+#define HIGH_PRIORITY_CLASS 0x00008000
```

Comment:
Looks like this renames `ABOVE_NORMAL_PRIORITY_CLASS` to `HIGH_PRIORITY_CLASS` but keeps the same value, meaning that it still acts as `ABOVE_NORMAL_PRIORITY_CLASS`. `HIGH_PRIORITY_CLASS` should actually be `0x00000080`.

But actually those definitions can probably be removed because the only reason they are here is that in some old versions of Windows headers `ABOVE_NORMAL_PRIORITY_CLASS` wasn't present. But `HIGH_PRIORITY_CLASS` should be present so no need to keep it at all.